app/static/src/app/components/sessions/SessionsPage.vue

```diff
@@ -2,15 +2,66 @@
   <div class="container">
     <div class="row">
       <h2>Sessions</h2>
-      <div> Coming soon - Sessions list!</div>
+    </div>
+    <template v-if="sessionsMetadata">
+      <div class="row fw-bold py-2">
+        <div class="col-3 session-col-header">Saved</div>
+        <div class="col-2 session-col-header">Label</div>
+        <div class="col-2 text-center session-col-header">Load</div>
+      </div>
+      <div class="row py-2" v-for="session in sessionsMetadata" :key="session.id">
+        <div class="col-3 session-col-value session-time">{{formatDateTime(session.time)}}</div>
+        <div class="col-2 session-col-value session-label" :class="session.label ? '' : 'text-muted'">
+          {{session.label || "--no label--"}}
+        </div>
+        <div class="col-2 text-center session-col-value session-load">
+          <a :href="sessionUrl(session.id)">
+            <vue-feather class="inline-icon brand" type="upload"></vue-feather>
```

Comment:
Ideally this would only be shown for sessions that are not the current session (so we'd still want to include the current session in the table I think, but disable loading by making this non-clickable and probably a different icon - perhaps CheckIcon or CheckCircleIcon?). I'm not sure where the session id is but might be easiest to manage if we add it to the store/sessions/state to make the logic here easy?

Reply:
Or we could just have some label indicating that it's the current session, and let people navigate to it, but that wouldn't reload, it would just navigate back to the main page?
In the mockups we had a special link back to the current session at the top of the page (I think there's a ticket for adding that) but nothing in particular to distinguish it in the list. 
SessionId is already in the store, so this should be easy to do.

Reply:
that option sounds great

Reply:
Could still use a different icon for 'loading' the current session. Maybe `home`?